scripts/ansible-todd-dev.yml

```diff
@@ -1,82 +1,106 @@
-
-  - name: Create ToDD directory
-    file: path={{ todd_dir }} state=directory mode=0755 owner=vagrant group=vagrant
-
-     # // TODO need to remove this dir first?
-  - name: Copy ToDD into VM
-    shell: cp -r /vagrant/* {{ todd_dir }}
-
-  - name: Set dir perms
-    shell: chown -R vagrant:vagrant {{ todd_dir }} && chmod -R 777 {{ todd_dir }}
-
+    - name: Set up docker repo
+      apt_repository:
+        repo: '{{ docker_repo }}'
+        state: present
+
+    - name: Install APT dependencies
+      apt:
+        name: "{{ item }}"
+        state: latest
+        update_cache: yes
+        force: yes  # Currently forcing because of a certificate error for docker-engine
+      with_items:
+      - git
+      - docker-engine
+
+    - name: Add vagrant user to docker group
+      user:
+        name: "vagrant"
+        groups: "docker"
+        append: yes
+
+    - name: Ensure docker service is restarted
+      service:
+        name: "docker"
+        state: restarted
+
+    - name: Download Go tarball
+      get_url:
+        url: "https://storage.googleapis.com/golang/go{{ gover }}.linux-amd64.tar.gz"
+        validate_certs: no
+        dest: "/tmp/go{{ gover }}.linux-amd64.tar.gz"
+        mode: 0770
+
+    - name: Unzip Go tarball
+      unarchive:
+        src: "/tmp/go{{ gover }}.linux-amd64.tar.gz"
+        dest: /usr/local
+        copy: no
+
+    - name: Set PATH
+      lineinfile:
+        dest: "/etc/environment"
+        state: present
+        regexp: "^PATH"
+        line: "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/local/go/bin:/home/vagrant/go/bin"
+
+    - name: Set GOPATH
+      lineinfile:
+        dest: "/etc/environment"
+        state: present
+        regexp: "^GOPATH"
+        line: "GOPATH=/home/vagrant/go"
+
+    - name: Set path for sudoers
+      lineinfile:
+        dest: "/etc/sudoers"
+        line: "Defaults  secure_path=\"/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin\""
+
+    - name: Create ToDD directory
+      file:
+        path: "{{ todd_dir }}"
+        state: directory
+        owner: vagrant
+        group: vagrant
+        mode: 0755
+
+       # // TODO need to remove this dir first?
+    - name: Copy ToDD into VM
+      copy:
+        src: "/vagrant/"
+        dest: "{{ todd_dir }}"
+
+    - name: Set dir perms
+      file:
+        src: "{{ todd_dir }}"
+        state: directory
+        recurse: yes
+        owner: "vagrant"
+        group: "vagrant"
+        mode: 0777
 
 - name: Prepping for ToDD
   hosts: all
   vars:
-    todd_dir: /home/vagrant/go/src/github.com/Mierdin/todd
+    todd_dir: '/home/vagrant/go/src/github.com/Mierdin/todd'
   tasks:
 
-  - name: Set GOPATH for Vagrant user profile
-    lineinfile: 
-      dest=/home/vagrant/.profile
-      state=present
-      regexp='^GOPATH'
-      line='GOPATH=/home/vagrant/go'
+    - name: Set GOPATH for Vagrant user profile
```

Comment:
fixed now..

On Mon, May 9, 2016 at 9:38 PM, Matt Oswalt <notifications@github.com>
wrote:

> In scripts/ansible-todd-dev.yml
> <https://github.com/Mierdin/todd/pull/23#discussion_r62601111>:
>
> >    tasks:
> >
> > -  - name: Set GOPATH for Vagrant user profile
> > -    lineinfile:
> > -      dest=/home/vagrant/.profile
> > -      state=present
> > -      regexp='^GOPATH'
> > -      line='GOPATH=/home/vagrant/go'
> > +    - name: Set GOPATH for Vagrant user profile
>
> The change in indentation is causing an error with - name: Print
> follow-up instructions. Please make sure these two tasks are indented
> identically.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Mierdin/todd/pull/23/files/ce2b2d563b0d8dffdbdcf8ea9833c03c6af2d00c#r62601111>
>



-- 
Twitter @mrlesmithjr <http://twitter.com/mrlesmithjr>
Blog EverythingShouldBeVirtual.com
Interested in blog sponsorship? Go to http://bit.ly/10Aj7Sq